python/src/tiff.rs

```diff
@@ -32,7 +32,9 @@ impl PyTIFF {
         let reader = store.into_async_file_reader(path);
 
         let cog_reader = future_into_py(py, async move {
-            let metadata_fetch = PrefetchBuffer::new(reader.clone(), prefetch).await.unwrap();
```

Comment:
Ideally we should remove all of these `unwrap`s, but we can leave that for the future.